assets/logging-cr.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: "logging.openshift.io/v1alpha1"
+kind: "ClusterLogging"
+metadata:
+  name: "instance"
+  namespace: "openshift-logging"
+spec:
+  managementState: "Managed"
+  logStore:
+    type: "elasticsearch"
+    elasticsearch:
+      nodeCount: 3
+      storage: {}
+      redundancyPolicy: "SingleRedundancy"
+      nodeSelector: 
+        node-role.kubernetes.io/infra: ""
+      resources:
+        request:
+          memory: 4G
```

Comment:
@thoraxe Did you get the `4G` from the docs? If so, it is not sufficient for a _real_ ES install. For whatever reason, the default is not working for me and I have to enter a value (this value works to bring up the pods in my cluster, and that is all I really need). I believe the default ES memory requirement is 16G.

Reply:
The default cluster installation uses m4.large which is not big enough for the default ES memory. The goal for beta was just "make things work" and not "fully explain all repercussions of whatever".